backend/service/martini/fcc.go

```diff
@@ -20,18 +22,37 @@ func (s *Service) FccFrnHandler() rest.Handle {
 			marshal.WriteJSON(w, r, httperror.WithInvalidRequest("missing filer_id parameter"))
 			return
 		}
+
+		id, err := db.ID(filerID)
+		if err != nil {
+			marshal.WriteJSON(w, r, httperror.WithInvalidRequest("invalid filer_id parameter").WithCause(err))
+			return
+		}
+
+		cached, err := s.db.GetFRNResponse(r.Context(), id)
+		if err == nil {
+			w.Header().Set(header.ContentType, header.ApplicationJSON)
+			w.Write([]byte(cached.Response))
+			return
+		}
+
```

Comment:
You could put a trace message here so that we know if an error occurred and cache was missed.